src/gui/osutils/nixutils/NixUtils.cpp

```diff
@@ -178,11 +184,14 @@ bool NixUtils::isCapslockEnabled()
 
 void NixUtils::registerNativeEventFilter()
 {
+#ifdef WITH_XC_X11
     qApp->installNativeEventFilter(this);
+#endif
```

Comment:
No need to wrap this, might cause bugs in the future if we add wayland events and forget to unwrap it